Toolset/palettes/standalone settings/revstandalonesettingsmacbehavior.livecodescript

```diff
@@ -7,12 +7,17 @@ on preOpenCard
    --   and replace with Intel.
    if tStandaloneSettingsA["MacOSX"] then
       delete variable tStandaloneSettingsA["MacOSX"]
-      put true into tStandaloneSettingsA["MacOSX x86-32"]
+      put true into tStandaloneSettingsA["MacOSX x86-64"]
    end if
    
    if tStandaloneSettingsA["MacOSX PowerPC-32"] then
       delete variable tStandaloneSettingsA["MacOSX PowerPC-32"]
-      put true into tStandaloneSettingsA["MacOSX x86-32"]
+      put true into tStandaloneSettingsA["MacOSX x86-64"]
+   end if
+   
+   if tStandaloneSettingsA["MacOSX x86-32"] then
+      put true into tStandaloneSettingsA["MacOSX x86-64"]
+      put false into tStandaloneSettingsA["MacOSX x86-32"]
    end if
```

Comment:
@livecodealex I think this `if` block has to be removed, because:
1. If an existing stack has the 32-bit checked, this will change to 64 bit without the user knowing that.
2. If you create a new stack, check both 32-bit and 64-bit, go to a different tab (e.g. "Windows") and then back to OSX tab, the 32-bit option will be unchecked.